src/main/java/lotto/domain/LottoMachine.java

```diff
@@ -0,0 +1,34 @@
+package lotto.domain;
+
+import java.util.List;
+
+public class LottoMachine {
+
+    private final List<Integer> winningNumbers;
+    private final int bonusNumber;
+    private final int LOTTO_LENGTH = 6;
```

Comment:
LOTTO_LENGTH는 상수로 두실려고 하신 것 같아요. 👀 

```suggestion
    private static final int LOTTO_LENGTH = 6;

    private final List<Integer> winningNumbers;
    private final int bonusNumber;
```